_config.yml

```diff
@@ -58,18 +48,7 @@ defaults:
       type: "resource_page"
       layout: "resource_page"
       sidebar: resources
-  -
-    scope:
-      path: "pages/about"
-    values:
-      type: "pages"
-      sidebar: about
-  -
-    scope:
-      path: "pages/contribute"
-    values:
-      type: "pages"
-      sidebar: contribute
```

Comment:
It was decided that the default `Contribute` button should be removed from the navbar.